tests/unittests/sources/test_ec2.py

```diff
@@ -945,11 +946,15 @@ def test_invalid_ipv4_ipv6_cidr_metadata_logged_with_defaults(self):
 
 class TestBuildNicOrder:
     @pytest.mark.parametrize(
-        ["macs_metadata", "macs", "expected"],
+        ["macs_metadata", "macs_to_nics", "default_nic_order", "expected"],
         [
-            pytest.param({}, [], {}, id="all_empty"),
+            pytest.param({}, {}, None, {}, id="all_empty"),
```

Comment:
Thanks for taking care of unit tests! For correctness, could we use the values of NicOrder?

```suggestion
            pytest.param({}, {}, NicOrder.MAC, {}, id="all_empty"),
            pytest.param(
                {}, {}, NicOrder.NIC_NAME, {}, id="all_empty_sort_by_nic_name"
            ),

```